test/unit/auxiliary/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+#
+# Distributed Linear Algebra with Future (DLAF)
+#
+# Copyright (c) 2018-2019, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+add_subdirectory(mc)
```

Comment:
Perhaps we can move this to the top CMakeLists.txt?

Reply:
About this comment and your next one, see https://github.com/eth-cscs/DLA-Future/pull/224#issuecomment-633860153

I've just opened an issue about this #236 (the title of the issue is not very straightforward, but it is about this).